### ISSUE TITLE
Handlers not being triggered

### ISSUE THREAD

**Issue by Timeraa** [MinnDevelopment/Java-DiscordRPC#29]
Hey,

So i already set up the presence etc everything works, but how do i catch an error/check if discord is even running?

**Comment by MinnDevelopment** [comment]
You can use the event handlers you pass to `Discord_Initialize` to determine when it connects or disconnects.

**Comment by Timeraa** [comment]
Okay so after some testing i got the Ready event to work but every other event is not working...

```JAVA
        handlers.errored = (err, err1) -> {
            System.out.println(err + "\n" + err1);
        };
        handlers.disconnected = (err, err1) -> {
            System.out.println(err + "\n" + err1);
        };
        handlers.ready = (user) -> {
            System.out.print(user);
        };
```

This is the code i am using for the events. I added it under the handlers declaration (initPresence())

if you want the full class (Note the events are not included):
[here](https://github.com/MCDiamondFire/DiamondFireUtilities/blob/master/src/main/java/dfutils/utils/rpc/PresenceHandler.java)

**Comment by MinnDevelopment** [comment]
Handlers not being triggered can only be caused by the official SDK. This seems related to discordapp/discord-rpc#220

**Comment by Timeraa** [comment]
I am using it on MacOS and i get this famous LRegistry error. Maybe its related to the events not being triggered?

**Comment by MinnDevelopment** [comment]
That error means it was unable to create the command used to start the app from within discord.

**Comment by MinnDevelopment** [comment]
I have proposed a fix for the disconnected event handler to the official SDK, it is currently waiting to be reviewed.

**Comment by MinnDevelopment** [comment]
This should be fixed in 2.0.2, feel free to open a new issue for other complications.
